### PR TITLE
Fix flaky tests

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: sdk/servicebus/Azure.Messaging.ServiceBus/tests/Diagnostics/EventSourceLiveTests.cs [hunk 1]
```diff
@@ -416,7 +416,7 @@ public async Task StoppingProcessorDoesNotLogTaskCanceledExceptions()
         {
             await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitioning: false, enableSession: false))
             {
-                await using var client = CreateNoRetryClient(10);
+                await using var client = CreateNoRetryClient();
```
Comment (JoshLove-msft) [inline sdk/servicebus/Azure.Messaging.ServiceBus/tests/Diagnostics/EventSourceLiveTests.cs:419]: Use the default which is 15 seconds.

**Review by jsquire — APPROVED** [review]

**Pull request merged**